Arpi/index.html

```diff
@@ -1,3 +1,19 @@
-<h1>Վեբ մեխանիկա</h1>
-<h2>Ի՞նչ է URL-ը</h2>
-<p>URL-ը առանցքային հասկացություն է, երբ խոսքը վերաբերում է ինտերնետին: Դա այն մեխանիզմն է, որն օգտագործվում է բրաուզերների կողմից համացանցում հրապարակված ցանկացած ռեսուրս ստանալու համար:</p>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Arpi</title>
+    <h1>Սյունիք</h1>
```

Comment:
<head> element can contain only meta datas, all semantic elements should be into body